scripts/edgecore_control_plane.sh

```diff
@@ -14,10 +14,10 @@ if [ -f "${DIR}/prerequisite_packages.sh" ]; then
 source "${DIR}/prerequisite_packages.sh"
 fi
 echo y | apt-get update
-rm -rf /root/.ssh/*
+rm -rf /root/.ssh/id_rsa  &&  rm -rf /root/.ssh/id_rsa.pub
```

Comment:
Could you put line 17 to line 20 put to key_gen() function.(like cloudcore_node.sh). Also check see if id_rsa and id_rsa.pub is exist.  If already exist, skip ("/dev/zero ssh-keygen -q -N "), if not exist, need to run "ssh-keygen" to generate id_rsa and id_rsa.pub.